.travis.yml

```diff
@@ -34,6 +34,7 @@ script:
       docker build -t kbclient .;
     fi
   - cd ../shared && npm i --global flow-bin@`tail -n1 .flowconfig` && flow
+  - cd ../protocol && npm i && gem install activesupport && make clean && make && git diff --quiet --exit-code
```

Comment:
Can this be done before `./test/run_tests.sh`?  If there's a bug in the new AVDLs or the AVDL compiler, then that build will fail.